### ISSUE TITLE
add transactions, note_to_payer and redirect_urls as props

### ISSUE THREAD

**Issue by liesislukas** [thinhvo0108/react-paypal-express-checkout#26]
transaction has many options and now props allow only `amount.total` and `amount.currency`, it should allow to submit full `transactions` object and also `note_to_payer`, `redirect_urls`, `intent`

new props:

- intent
- payer
- transactions
- note_to_payer
- redirect_urls

or just options
https://developer.paypal.com/docs/api/payments/

```json
{
  "intent": "sale",
  "payer": {
  "payment_method": "paypal"
  },
  "transactions": [
  {
    "amount": {
    "total": "30.11",
    "currency": "USD",
    "details": {
      "subtotal": "30.00",
      "tax": "0.07",
      "shipping": "0.03",
      "handling_fee": "1.00",
      "shipping_discount": "-1.00",
      "insurance": "0.01"
    }
    },
    "description": "The payment transaction description.",
    "custom": "EBAY_EMS_90048630024435",
    "invoice_number": "48787589673",
    "payment_options": {
    "allowed_payment_method": "INSTANT_FUNDING_SOURCE"
    },
    "soft_descriptor": "ECHI5786786",
    "item_list": {
    "items": [
      {
      "name": "hat",
      "description": "Brown hat.",
      "quantity": "5",
      "price": "3",
      "tax": "0.01",
      "sku": "1",
      "currency": "USD"
      },
      {
      "name": "handbag",
      "description": "Black handbag.",
      "quantity": "1",
      "price": "15",
      "tax": "0.02",
      "sku": "product34",
      "currency": "USD"
      }
    ],
    "shipping_address": {
      "recipient_name": "Brian Robinson",
      "line1": "4th Floor",
      "line2": "Unit #34",
      "city": "San Jose",
      "country_code": "US",
      "postal_code": "95131",
      "phone": "011862212345678",
      "state": "CA"
    }
    }
  }
  ],
  "note_to_payer": "Contact us for any questions on your order.",
  "redirect_urls": {
  "return_url": "https://example.com/return",
  "cancel_url": "https://example.com/cancel"
  }
}'
```

**Comment by liesislukas** [comment]
i think this is solved in https://github.com/thinhvo0108/react-paypal-express-checkout/pull/22

**Comment by interpegasus** [comment]
Hi, 
I would like to know if there is a prop to pass an array of items in the shopping cart to Paypal? 
#22
Thank you.

Reference:
https://developer.paypal.com/docs/classic/express-checkout/integration-guide/ECCustomizing/#provide-line-item-details

**Comment by thinhvo0108** [comment]
#22 has been merged, and latest version @1.0.5 has just been updated on NPM, too, thanks!
